src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -21,66 +20,113 @@
 @Service
 public class EmailService implements IEmailService {
 
-    @Value("${EMAIL_SUBJECT}")
-    private String emailSubject;
-
-    @Value("${EMAIL_SENDER_APP_ID}")
-    private String originatingAppId;
-
-    @Value(("${EMAIL_SUBMITTED_EXTERNAL_TEMPLATE_MESSAGE_TYPE}"))
-    private String submittedCustomerEmailType;
-
+    private EmailConfig emailConfig;
     private ApiLogger logger;
-    private ICompanyProfileService companyProfileService;
     private KafkaEmailClient kafkaEmailClient;
     private Supplier<LocalDateTime> dateTimeSupplier;
 
     @Autowired
     public EmailService(
+            EmailConfig emailConfig,
             ApiLogger logger,
-            ICompanyProfileService companyProfileService,
             KafkaEmailClient kafkaEmailClient,
             Supplier<LocalDateTime> dateTimeSupplier
     ) {
+        this.emailConfig = emailConfig;
         this.logger = logger;
-        this.companyProfileService = companyProfileService;
         this.kafkaEmailClient = kafkaEmailClient;
         this.dateTimeSupplier = dateTimeSupplier;
     }
 
     @Override
     public void sendObjectionSubmittedCustomerEmail(
             Objection objection,
+            String companyName,
             String requestId
     ) throws ServiceException {
-        String companyNumber = objection.getCompanyNumber();
 
-        CompanyProfileApi companyProfile = companyProfileService.getCompanyProfile(companyNumber, requestId);
-
-        String companyName = companyProfile.getCompanyName();
         String emailAddress = objection.getCreatedBy().getEmail();
-        Map<String, Object> data = new HashMap<>();
+        Map<String, Object> data = constructEmailDataMap(
+                companyName,
+                objection,
+                objection.getCreatedBy().getEmail());
 
-        String subject = emailSubject.replace("{{ COMPANY_NUMBER }}", companyNumber);
-        data.put("subject", subject);
-        data.put("to", emailAddress);
-        data.put("company_name", companyName);
-        data.put("company_number", companyNumber);
-        data.put("objection_id", objection.getId());
-        data.put("reason", objection.getReason());
-        data.put("attachments", objection.getAttachments());
+        EmailContent emailContent = constructEmailContent(EmailType.CUSTOMER,
+                emailAddress, data);
+
+        logger.debugContext(requestId, "Calling Kafka client to send customer email");
+        kafkaEmailClient.sendEmailToKafka(emailContent);
+        logger.debugContext(requestId, "Successfully called Kafka client");
+    }
+
+    @Override
+    public void sendObjectionSubmittedDissolutionTeamEmail(
+            CompanyProfileApi companyProfile,
```

Comment:
Looks like we only need the jurisdiction and companyName out of companyProfile, I think it would be better to just pass those 2 strings in as args rather than a CompanyProfileApi, it would be consistent with the customer email args as well.